subprojects/icu.wrap

```diff
@@ -6,6 +6,7 @@ source_hash = 67a7e6e51f61faf1306b6935333e13b2c48abd8da6d2f46ce6adca24b1e21ebf
 patch_directory = icu
 
 [provide]
+ICU = icuuc_dep
```

Comment:
This is outright broken, the cmake.org published FindICU.cmake module (not officially part of the icu project) exports ICU::uc, as well as ICU::io, ICU::i18n, so in order to use cmake lookups you need to pass a `modules:` kwarg, and there's a fairly good chance the user didn't want icu-uc specifically.

Reply:
Hmm, they indeed pass a component: https://github.com/harfbuzz/harfbuzz/blob/main/meson.build#L96. Our `[provide]` mechanism cannot deal with components, so I guess there is no solution here :/